pkg/get/get.go

```diff
@@ -195,7 +196,8 @@ https://storage.googleapis.com/kubernetes-release/release/{{.Version}}/bin/{{$os
 			Version:     "v0.9.0",
 			URLTemplate: `https://github.com/ahmetb/kubectx/releases/download/{{.Version}}/kubectx`,
```

Comment:
Theoretically this is now wrong as a URL if you are downloading the archive?

How about leaving this one as it is, and adding helm3 to the list of "get" tools instead?

Reply:
helm added 
![image](https://user-images.githubusercontent.com/627278/85117897-c246d480-b249-11ea-8f17-f6309a0cb76d.png)